### ISSUE TITLE
BIA Review of MTD, RTO, WRT, and RPO

### ISSUE THREAD

**Issue by gary-fallon** [department-of-veterans-affairs/va.gov-team#53063]
## Description
Review and revise the following, as necessary.

- Maximum Tolerable Downtime (MTD)
- Recovery Time Objective (RTO)
- Work Recovery Time (WRT)
- Recovery Point Objective (RPO)

The VA.gov availability is *moderate*. This designation sets a maximum of 72 hours for the RTO and WRT total, and 24 hours for RPO.

Currently, times and objectives are defined as 6 hours. It is important that these values are as accurate as possible for all the listed services in Table 5. These services should also support any items listed in Tables 3.1 and 3.2, align with the platform architecture, and be proven out in the Business Continuity Plan (BCP) and Disaster Recovery Plan (DRP).

For example, EKS vets-api follows one workflow and has one set of values, while the legacy vets-website has a different workflow with a different set of values.

## Tasks
The following tasks must be reviewed and completed as necessary.
- [x] 3.3.2 Maximum Tolerable Downtime (MTD)
  - [x] The services in Table 5 do not appear complete.
- [x] 3.3.3 Recovery Time Objective (RTO)
- [x] 3.3.4 Work Recovery Time (WRT)
- [x] 3.3.5 Recovery Point Objective (RPO)
- [x] 3.3.6 Determine MTD Gap (MTD – RTO) 
- [x] 3.3.7 Gap Recommendation 
- [x] Update the document to reflect any new RTO, WRT, RPO, and MTD values.
  - [x] Execute Summary table (unlabeled)
  - [x] Table 3 
  - [x] Table 11

## Acceptance
The following tasks must be completed before this issue can be considered done.
- [x] Review for accuracy and completeness.

**Comment by little-oddball** [comment]
@alyssagallion - what might be best here is having Jesse and Kyle as well as Nate and Eric... I believe a fair amount of this information would be provided by Jesse/Kyle but Nate/Eric might have additional thoughts. I see this as also probably being a conversation that Gary gets pulled into to add context/color.

**Comment by little-oddball** [comment]
SA: This ticket is a child of the epic below which has the BIA, etc.: https://github.com/department-of-veterans-affairs/va.gov-team/issues/53055

**Comment by alyssagallion** [comment]
Assigning this ticket to @jhouse-solvd per our Slack conversation and meeting today.

**Comment by alyssagallion** [comment]
During our Gap Analysis, we realized that the System Security Plan (SSP) was incorrect. We have since updated it, presented it to the ISSO and Ray Wang is now reviewing the associated tickets he needs to approve. Once those are reviewed and approved, @ScottCutlip will continue the work on SSP. 

SSP is the dependency. Until approved, all BIA work is on hold.

**Comment by jhouse-solvd** [comment]
Please [see the parent epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/53055#issuecomment-1589591963) for a recent update. 

Moving to review.

**Comment by jhouse-solvd** [comment]
The most recent info has been included in the BIA. 

Please [see the parent epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/53055#issuecomment-1589591963) for future updates. 

Closing.
